app/admin.py

```diff
@@ -4,7 +4,7 @@
 from django.forms import Textarea
 from django.http import HttpRequest, HttpResponseRedirect
 from django.utils import timezone as tz
-from rangefilter.filter import DateRangeFilter
+from rangefilter.filters import DateRangeFilter
 
```

Comment:
um did they change the API? How was this ever working? 👀

Reply:
ah nvm. https://github.com/silentsokolov/django-admin-rangefilter/releases/tag/v0.8.0

Reply:
Yeah, it's part of the bump 😆